### ISSUE TITLE
Use native Object.assign instead of xtend

### ISSUE THREAD

**Issue by jimmywarting** [apostrophecms/sanitize-html#257]
[xtend](https://github.com/Raynos/xtend) can easily be replaced with [Object.assign](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign)

This reduce the dependencies and possible also avoid duplicated versions where some package don't use the same version range for the xtend package

```js
// immutable
Object.assign({}, a, b)

// mutable
Object.assign(a, b)
```

**Comment by abea** [comment]
We'd be happy to review a PR for this.

**Comment by abea** [comment]
Closed by #365
